.gitignore

```diff
@@ -5,3 +5,4 @@ npm-debug.log
 travis_phantomjs
 .DS_Store
 .vscode
+.samples.config.json
```

Comment:
I think we can remove:
 - `travis_phantomjs`
 - `.samples.config.json` (See below)

It might make sense to include this file with some default config. For example:
```
{
  issuer: 'https://{yourOktaDomain}.com/oauth2/default',
  client_id: '{client_id}',
  ...
}
```